c2corg_ui/templates/outing/view.html

```diff
@@ -45,6 +45,8 @@
   });
 </%block>
 
+<div class="ng-hide">{{outingUsers = ${outing['associations']['users']} }}</div>
```

Comment:
I'm a bit uncomfortable with that way of doing. Sounds a bit hackish to set global angular variables.
Can't we directly pass the associated users list as a directive attribute (protected-url-btn?) somewhere?